packages/paste-core/components/label/src/Label.tsx

```diff
@@ -15,10 +15,24 @@ export interface LabelProps extends React.LabelHTMLAttributes<HTMLLabelElement>,
   marginBottom?: 'space0';
   required?: boolean;
   variant?: LabelVariants;
+  i18nRequiredLabel?: string;
 }
 
 const Label = React.forwardRef<HTMLLabelElement, LabelProps>(
-  ({as = 'label', marginBottom, required, disabled, children, variant, element = 'LABEL', ...props}, ref) => {
+  (
+    {
+      as = 'label',
+      marginBottom,
+      required,
+      disabled,
+      children,
+      variant,
+      element = 'LABEL',
+      i18nRequiredLabel = '',
```

Comment:
question: is it safe to say that if the label is rendered as a label `i18nRequiredLabel` should be empty by default, but if it is rendered as legend `i18nRequiredLabel` should be `(required)` by default. Is that an over simplification?

Reply:
Oh! thats exactly correct.

I just had it default to nothing and then checkboxGroup and radioGroup set it to '(required)'.